test-status/api.js

```diff
@@ -279,4 +282,27 @@ exports.installApiRouter = (app, db) => {
 
         return response.end();
       });
+
+  const buildCop = app.route('/v0/build-cop');
+  buildCop.use(require('express').json());
+  buildCop.use((request, response, next) => {
+    if (!('accessToken' in request.body) ||
+        request.body.accessToken != process.env.BUILD_COP_UPDATE_TOKEN) {
+      app.log(`Refused a request to ${request.originalUrl} from ${request.ip}`);
+      response.status(403).end('Access token missing or not authorized');
+    } else {
+      next();
+    }
+  });
+
+  buildCop.post('/update', async (request, response) => {
+    if (!('username' in request.body)) {
```

Comment:
what if key exists but field value is empty string? is that possible?

Reply:
You're right, I added a check for this, just in case the cron job accidentally tries to update an empty string